test/fiddle/test_import.rb

```diff
@@ -130,6 +130,8 @@ def test_sizeof()
         name = $1.sub(/P\z/,"*").gsub(/_(?!T\z)/, " ").downcase
         type_name = name
       end
+      type_name = "unsigned #{$1}" if type_name =~ /u(long|short|char)/
```

Comment:
```suggestion
      type_name = "unsigned #{$1}" if type_name =~ /\Au(long|short|char)/
```

Do we need to add `int` too here?

Reply:
We might have a missing test.  It seems like we need `int`, but no tests fail.  I will investigate